frontend/javascripts/admin/admin_rest_api.ts

```diff
@@ -339,13 +339,17 @@ export function updateTaskType(taskTypeId: string, taskType: APITaskType): Promi
 
 // ### Teams
 export async function getTeams(): Promise<Array<APITeam>> {
-  const teams = await Request.receiveJSON("/api/teams");
+  const teams = await Request.receiveJSON("/api/teams", {
+    doNotInvestigate: true,
```

Comment:
In `share_modal_view.tsx`, `showErrorToast: false` is used twice for this use case. However, I think your solution makes more sense, maybe you can unify the behavior for the share_model code as well :)

Reply:
To follow up with our in-person convo: I could remove the request parameters from `share_modal_view.tsx` by checking whether the user is logged in. However, the `doNotInvestigate` params in this module are quite handy for the team related selection UIs (`PermissionsAndTeamsModalView` and `TeamSelectionComponent`) which I didn't want to adapt for this hot-fix PR.